### PR TITLE
Mage_Customer - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
#681 DOC blocks only

### CODE REVIEW

File: app/code/core/Mage/Customer/Block/Account/Dashboard/Info.php [hunk 1]
```diff
@@ -34,11 +34,22 @@
 
 class Mage_Customer_Block_Account_Dashboard_Info extends Mage_Core_Block_Template
 {
+    /**
+     * @var Mage_Newsletter_Model_Subscriber
+     */
+    public $_subscription;
```
Comment (kkrieger85) [inline app/code/core/Mage/Customer/Block/Account/Dashboard/Info.php]: Suggestion: make class attribute private

There's no need to make this attribute accessible from outside (if so, please reference)

File: app/code/core/Mage/Customer/Block/Account/Dashboard/Newsletter.php [hunk 2]
```diff
@@ -34,6 +34,14 @@
 
 class Mage_Customer_Block_Account_Dashboard_Newsletter extends Mage_Core_Block_Template
 {
+    /**
+     * @var Mage_Newsletter_Model_Subscriber
+     */
+    public $_subscription;
```
Comment (kkrieger85) [inline app/code/core/Mage/Customer/Block/Account/Dashboard/Newsletter.php]: Suggestion: make class attribute private

There's no need to make this attribute accessible from outside (if so, please reference)

File: app/code/core/Mage/Customer/Model/Convert/Parser/Customer.php [hunk 3]
```diff
@@ -442,115 +455,6 @@ protected function _getCustomerGroupCode($customer)
         }
     }
 
-   /* ########### THE CODE BELOW IS NOT USED ############# */
```
Comment (tmotyl) [inline app/code/core/Mage/Customer/Model/Convert/Parser/Customer.php:445]: lets keep this comment or remove everything below it.

Comment (sreichel) [reply app/code/core/Mage/Customer/Model/Convert/Parser/Customer.php:445]: Old code is deleted, isn't it?

Comment (sreichel) [reply app/code/core/Mage/Customer/Model/Convert/Parser/Customer.php:445]: Yes, it is :)

Comment (tmotyl) [reply app/code/core/Mage/Customer/Model/Convert/Parser/Customer.php:445]: not all. " public function parse()" is not deleted

Comment (sreichel) [reply app/code/core/Mage/Customer/Model/Convert/Parser/Customer.php:445]: @tmotyl this was a code block/mehtod "OLD_something()" ... this has been deleted. All other dead code is untouched and will be remoded later. Please accept for now.

Comment (tmotyl) [reply app/code/core/Mage/Customer/Model/Convert/Parser/Customer.php:445]: ok

**Comment by tmotyl** [comment]
I have resolved the conflict, please @colinmollenhour revote so we can merge the beast.

**Comment by tmotyl** [comment]
Thank you! Im so happy we are moving this forward!!

**Pull request merged**